README.md

```diff
@@ -113,24 +97,9 @@ This file will then need to be referenced on the command line using the
 `--variables` option.
 
 ### Run the tests
-Tests are run using the command line. Below are a couple of examples of running the tests:
-
-To run all of the desktop tests against the default environment, which is the add-ons development environment:
-
-```bash
-$ py.test --driver Firefox --variables api.json --variables paypal.json tests/desktop
-```
-
-To run against a different environment, pass in a value for `--base-url`, like so:
-
-```bash
-$ py.test --base-url https://addons.allizom.org --driver Firefox --variables api.json --variables paypal.json  tests/desktop
-```
 
-The pytest plugin that we use for running tests has a number of advanced
-command line options available. To see the options available, run
-`py.test --help`. The full documentation for the plugin can be found
-[here][pytest-selenium].
+[Install Tox](https://tox.readthedocs.io/en/latest/install.html)
+Run `tox`
```

Comment:
Running `tox` will fail several tests because the variables file is not specified, so I think it's important to include it. My suggestion should 'just work' though.